src/Microsoft.DotNet.Cli.Utils/project.json

```diff
@@ -2,7 +2,13 @@
   "version": "1.0.0-preview3-*",
   "buildOptions": {
     "keyFile": "../../tools/Key.snk",
-    "warningsAsErrors": true
+    "warningsAsErrors": true,
+    "compile": {
+      "include": [
+        "**/*.cs",
+        "../../src/dotnet/DotnetFiles.cs"
```

Comment:
Actually, I have an idea to try and remove this file. I will add an optional parameters to the resolvers called ridCap that the tests can use. Then, the test themselves can include the DotnetFiles to read the .version that gets copied to their output. I will try this out.